ant-evm/src/data_payments.rs

```diff
@@ -77,6 +77,21 @@ impl ProofOfPayment {
             .any(|(_, quote)| quote.has_expired())
     }
 
+    /// Returns all quotes by given peer id
+    pub fn quotes_by_peer(&self, peer_id: &PeerId) -> Vec<&PaymentQuote> {
```

Comment:
Should this also check that the quote is valid for the same peer id?

Reply:
I think this is caller responsibility? The name doesn't suggest a check is there